kedro-airflow/tests/test_plugin.py

```diff
@@ -23,7 +23,6 @@ def test_create_airflow_dag(mocker, cli_runner, metadata):
     result = cli_runner.invoke(commands, ["airflow", "create"], obj=metadata)
 
     assert result.exit_code == 0
-    assert str(dag_file) in result.output
```

Comment:
This line is removed because the `result.output` shows the full path in Linux while it only shows the relative path in Windows and failing the test.

The following code tests the existence of the file and tries to open it, so it has the same purpose of this assertion.